cli/turbo.json

```diff
@@ -2,20 +2,47 @@
   "$schema": "../docs/public/schema.json",
   "extends": ["//"],
   "pipeline": {
+    "test:go": {
+      "dependsOn": ["build"]
+    },
+    "test:rust": {
+        "dependsOn": ["build"]
+    },
     "build": {
-      "outputs": ["turbo", "turbo.exe"]
+      "outputs": ["turbo", "turbo.exe", "../target/**/*"]
     },
-
-    "e2e": {
-      "outputs": [],
+    "lint:go": {
       "inputs": [
         "cli/**/*.go",
         "cli/go.mod",
-        "cli/go.sum",
-        "cli/scripts/e2e/e2e.ts",
-        "shim/**/*.rs"
+        "cli/go.sum"
+      ]
+    },
+    "lint:rust": {
+      "inputs": [
+        "../crates/**/*.rs"
       ]
     },
+    "lint": {
+        "dependsOn": ["lint:go", "lint:rust"]
+    },
+    "test": {
+        "dependsOn": ["test:go", "test:rust"]
```

Comment:
always contentious, but could depend on `lint` here if we want to do a quick pass before incurring the test cost